scripts/assets/test_file_exemptions.textproto

```diff
@@ -652,6 +652,7 @@ exempted_file_path: "domain/src/main/java/org/oppia/android/domain/state/StateDe
 exempted_file_path: "domain/src/main/java/org/oppia/android/domain/state/StateGraph.kt"
 exempted_file_path: "domain/src/main/java/org/oppia/android/domain/state/StateList.kt"
 exempted_file_path: "domain/src/main/java/org/oppia/android/domain/testing/oppialogger/loguploader/FakeLogUploader.kt"
+exempted_file_path: "domain/src/main/java/org/oppia/android/domain/testing/oppialogger/logscheduler/FakeLogScheduler.kt"
```

Comment:
Added tests for FakeLogScheduler and the configurationsModule. 
Apart from these two, all are interfaces except the workerFactory -- for which we can't really have a test, since we'd have to replicate workerParameters which we can't because the parameter values are scoped and hence not possible to re-create for tests.